app/models/event.rb

```diff
@@ -2,53 +2,83 @@ class Event < ActiveRecord::Base
   before_save :calc_duration
 
   attr_accessible :title, :description, :category, :course_id, :duration, :start_time, :end_time, :instructor, :location, :status, :timecard_ids, :person_ids, :comments
- 
-  def end_date_cannot_be_before_start
-    if ((!end_time.blank?) and (!start_time.blank?)) and end_time < start_time
-      errors.add(:end_time, "must be after the start, unless you are the Doctor")
-    end
-  end
+
   def end_date_cannot_be_before_start
```

Comment:
Could this be changed to `end_time_cannot_be_before_start` (here and in the `validate` line)?  I'd find it more readable to have this lower in the file, among the other methods, probably in the `private` section.